lib/ditto.ts

```diff
@@ -85,7 +120,7 @@ const setupOptions = () => {
 
 const executeCommand = async (
   command: Command | "none",
-  options: string[]
+  options: any
```

Comment:
`any` here because we don't know what flags are going to be passed in from the user. Probably possible to improve?

Reply:
I wonder if this type is always guaranteed to be an object? In which case it could be `Record<string, string>` or something along those lines. Leaving this as `any` for the moment doesn't seem like a big deal.